### ISSUE TITLE
[BUG] Xamarin.forms ABI break issue with X.F 4.4.0 on tizen

### ISSUE THREAD

**Issue by myroot** [mono/SkiaSharp#1167]
**Description**
 I got trouble with Xamarin.forms 4.4.0 on Tizen, some API signature was changed
 https://docs.microsoft.com/en-us/xamarin/xamarin-forms/release-notes/4.4/4.4.0-api
```
System.TypeLoadException: Method 'LoadImageAsync' in type 'SkiaSharp.Views.Forms.SKImageSourceHandler' from assembly 'SkiaSharp.Views.Forms, Version=1.68.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```
 So, need to update referenced Xamarin.forms package version up to 4.4.0

**Code**
 None

**Expected Behavior**
 Working

**Actual Behavior**
 Got an exception 
``` 
System.TypeLoadException: Method 'LoadImageAsync' in type 'SkiaSharp.Views.Forms.SKImageSourceHandler' from assembly 'SkiaSharp.Views.Forms, Version=1.68.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```
**Basic Information**

- Version with issue:  1.68.2-preview.39 with Xamarin.forms 4.4.0 
- Last known good version:  good with Xamarin.forms 4.2.0 
- IDE:  
- Platform Target Frameworks: 
  - Tizen:  tizen40  

**Screenshots**

**Reproduction Link**

**Comment by myroot** [comment]
It fixed by this PR https://github.com/mono/SkiaSharp/pull/1117

**Comment by mattleibow** [comment]
@myroot did you confirm this?

**Comment by myroot** [comment]
@mattleibow yes, #1117 is fixing this issue

**Comment by mattleibow** [comment]
Merged in #1117

**Comment by rusmozhaik** [comment]
@mattleibow Could you please release new preview version with these changes?

**Comment by mattleibow** [comment]
I have published these yesterday, v1.68.2-preview.43: https://www.nuget.org/packages/SkiaSharp/1.68.2-preview.43

Let me know if this is working for you.

**Comment by rusmozhaik** [comment]
@mattleibow thanks a lot! `TypeLoadException` is fixed now
